js/misc/paygManager.js

```diff
@@ -77,6 +77,7 @@ var PaygManager = GObject.registerClass({
         'enabled-changed': { param_types: [GObject.TYPE_BOOLEAN] },
         'expiry-time-changed': { param_types: [GObject.TYPE_INT64] },
         'initialized': {},
+        'rate-limit-end-time-changed':  { param_types: [GObject.TYPE_INT64] },
```

Comment:
Should this be `TYPE_UINT64`?

Reply:
And if it should, I guess `expiry-time-changed` should be too

Reply:
That's right, good catch!